test/integration/members.test.js

```diff
@@ -44,8 +45,9 @@ describe('Members', function () {
           expect(res).to.have.status(200)
           expect(res.body).to.be.a('object')
           expect(res.body.message).to.equal('Members returned successfully!')
-          expect(res.body.members).to.be.a('array')
-          expect(res.body.members[0].isMember).to.eql(true)
+          expect(res.body.members.newMembers).to.be.a('array')
+          expect(res.body.members.oldMembers).to.be.a('array')
+          expect(res.body.members.oldMembers[0].isMember).to.eql(true)
```

Comment:
`newMembers` would also be having `isMember` true, right?

Reply:
Right now 'Yes'. 
All are members, plus their userType value would be 'new'.

Reply:
Can we assert on that as well?

Reply:
This needs discussion based on my questions raised above.